modules/custom/az_dam/config/quickstart/filter.format.basic_html.yml

```diff
@@ -0,0 +1,74 @@
+uuid: d0c85e92-3483-4935-9664-ec2b98c1af5c
+langcode: en
+status: true
+dependencies:
+  module:
+    - bootstrap_utilities
+    - editor
+    - ib_dam_wysiwyg
+    - media
+_core:
+  default_config_hash: vxZvKWwZReEd1yDpL5Mmxyj5-8fRTjHAGqJaIaq7toA
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.